src/components/AccountDrawer/MiniPortfolio/index.tsx

```diff
@@ -61,14 +62,7 @@ const PageWrapper = styled.div`
   flex: 1;
 `
 
-interface Page {
```

Comment:
These components are fairly unrelated, to avoid potentially causing a regression in the Mini Portfolio, I'd suggest **against** refactoring this simple interface to be used for the Explore page as well.